scripts/spectrum-vars.js

```diff
@@ -128,9 +130,7 @@ const processTypography = async (
     const result = await processCSSData(data, identifier, from, usedVariables);
     fs.writeFileSync(dstPath, result, 'utf8');
 
-    const fontPath = path.resolve(
```

Comment:
These resolve + requires were resulting in a double drive prefix being added to all the files. We'll need to document this in our scripting best practices for the team.